src/theme/typography.ts

```diff
@@ -1,8 +1,9 @@
+import type { TypographyOptions } from '@mui/material/styles/createTypography';
```

Comment:
was receiving type errors when using `typography.js` after converting `protocolTheme` to a typescript file